src/components/Tabs/TabItem.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+
+import { StyledTabItem } from "./TabItem.styled";
+
+const TabItem = (props) => {
+  const [item, setItem] = useState("Profile");
+
+  const handleClick = (e) => {
+    setItem(e.target.innerHTML);
```

Comment:
Jest to rozwiązanie vanilla JS, a my chcemy React :D
Trzymaj w `state` numer lub identyfikator dziecka - i nich to on decyduje, co ma być wyświetlane.
Zobacz, że masz coś takiego jak `eventKey="home"` - możesz to wykorzystać.
Jednocześnie w state przechowywałbym, który `eventKey` jest aktywny - dzięki tej informacji będziesz wiedział jak "ostylować" element aktywny.